src/main/java/edu/ksu/canvas/attendance/exception/AttendanceAssignmentException.java

```diff
@@ -0,0 +1,46 @@
+package edu.ksu.canvas.attendance.exception;
+
+import edu.ksu.canvas.attendance.entity.AttendanceAssignment;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'edu.ksu.canvas.attendance.entity.AttendanceAssignment'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.ome.ksu.edu/coding_rules#rule_key=squid%3AUselessImportCheck)